File: 01.Tech_Module/Software_Technologies_July_2017/31. ReTake Exam/JS AnimeList/models/Product.js
const mongoose = require('mongoose');

let productSchema = mongoose.Schema({
    //TODO: Implement me ...
    priority: { type: Number, required: true },
    name: { type: String, required: true },
    quantity: { type: Number, required: true },
    status: { type: String, required: true  }
});

let Product = mongoose.model('Product', productSchema);

module.exports = Product;

File: 01.Tech_Module/Software_Technologies_-_Jul_2017/31. ReTake Exam/C# AnimeList/ShoppingList/Models/Product.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace ShoppingList.Models
{
    public class Product
    {
        //TODO: Implement me ...
        [Key]
        public int Id { get; set; }

        [Required]
        [AllowHtml]
        public string Priority { get; set; }

        [Required]
        [AllowHtml]
        public string Name { get; set; }

        [Required]
        [AllowHtml]
        public string Quantity { get; set; }

        [Required]
        public string Status { get; set; }
    }
}